src/components/TodoFromComponents/TodoFrom.module.css

```diff
@@ -0,0 +1,26 @@
+.TodoForm {
+    margin-bottom: 20px;
+  }
+  
+  .todo-input {
+    width: calc(100% - 18px);
+    padding: 8px;
+    font-size: 16px;
+    margin-bottom: 10px;
+    border-radius: 5px;
+    border: 1px solid #4E8CAB;
+  }
+  
+  .todo-btn {
+    width: 100%;
+    padding: 8px;
+    font-size: 16px;
+    background-color: #007EA7;
+    color: white;
+    border: none;
+    border-radius: 5px;
+    cursor: pointer;
+    margin-top: 10px;
+  }
+  
```

Comment:
in module css we name the style classes like a js variable
for example todoBtn
then, when you use it in you component you can do
styles.todoBtn insted of styles["todo-btn"]